app/assets/stylesheets/components/_repo-list.scss

```diff
@@ -1,8 +1,9 @@
 .repo-list {
   display: flex;
   flex-wrap: wrap;
-  margin-right: -$base-spacing;
   padding-top: 1.25rem;
+  max-width: 100rem;
```

Comment:
Isn't this going to be around ~1200px? I'd be worried that this is still too wide without some padding on the sides if the screen is near whatever 100rem comes out to.